designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala

```diff
@@ -146,6 +146,19 @@ extends Repository[F] with EspTables with CommentActions with ProcessActionRepos
     run(query.result.map(_.toSet))
   }
 
+  def getLastStateActionType(processId: ProcessId): F[Option[ProcessActionType]] = {
```

Comment:
Let's have a more general method `getLastFinishedActionTypes(ProcessId, Set[ActionType])` and move this upper to the place of usage, that:
1) will be clear why this set is important
2) will be the chance that we will use it in more places